kokoro_build.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# Fail on any error.
+set -e
+# Display commands being run.
+set -x
+
+# Update compilers to use gcc/g++/cpp-6 as default
+sudo apt-get update
+sudo apt-get install -y gcc-6
```

Comment:
GCC 6 is fine?
I was getting ICE with 6.3.x

Reply:
Yes it's fine for Ubuntu, which is used by Kokoro. We can change to GCC 7 if you'd prefer.

Reply:
If 6 works we can keep it there for this case.

Reply:
Sounds good.